### ISSUE TITLE
【2.0 API文档内测】BatchNorm2D文档问题

### ISSUE THREAD

**Issue by lwbmowgli** [PaddlePaddle/Paddle#28949]
文档问题描述：
paddle.nn.BatchNorm2D文档评估

API文档描述是否清晰？
中文：5分，整体描述比较清楚，在描述中给出了论文参考；
英文：3分，显示出来的是Markdown格式的，影响阅读体验，

参数说明是否清晰
中文：,5分，描述较清晰；
英文：5分

返回/形状说明是否清晰
中文：5分，形状描写的很清楚；
英文：5分

示例代码是否有效？
中文：4分，示例代码很简单，好像有点过于简单了；
英文：4分，

中英文内容是否一致？
一致

其他
,英文文档中，代码示例后面的描述是继承了Layer的一些方法 ，未隐藏这部分信息。

整体评价
4.5分，整体描述得非常清晰，中文文档描述的很好，英文的存在显示问题。
